addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -327,46 +327,45 @@ var Wysiwyg = Widget.extend({
             $dropDownToToggle.find(".dropdown-toggle").dropdown("toggle");
         }
     },
-    async initColorPicker(colorPickerid, setCommandId, unsetCommandId, $dropdownNode) {
-        if (!this.colorPickers[colorPickerid]) {
+    async initColorPicker($dropdownNode, setCommandId, unsetCommandId) {
+        if (!$dropdownNode.hasClass("colorpicker-initalized")) {
+            $dropdownNode.addClass("colorpicker-initalized");
             // Init the colorPalete for this color picker Dropdown.
-            this.colorPickers[colorPickerid] = new ColorPaletteWidget(this, {});
+            const colorpicker = new ColorPaletteWidget(this, {});
```

Comment:
In case I have multiple toolbar instance, I need multiple colorpicker to be placed in the corresponding dropdown.
A colorpicker instance is only created the first time a dropdown is open.